jetty-xml/src/test/resources/org/eclipse/jetty/xml/configureWithElements.xml

```diff
@@ -87,25 +87,33 @@
   <Put name="Float" type="Float">2.3</Put>
   <Put name="Env"><Env name="HOME"/></Put>
 
+  <New>
+    <Id>testId</Id>
+    <Class>org.eclipse.jetty.xml.TestConfiguration</Class>
+    <Set name="testString">nested</Set>
+  </New>
+
   <Set name="nested">
-    <New>
-      <Class>org.eclipse.jetty.xml.TestConfiguration</Class>
-      <Set name="testString">nested</Set>
+    <Ref>
+      <Id>testId</Id>
       <Set name="nested">
         <New class="org.eclipse.jetty.xml.TestConfiguration">
         </New>
       </Set>
-    </New>
+    </Ref>
   </Set>
 
-  <Call name="call">
+  <Call>
+    <Name>call</Name>
   </Call>
 
-  <Call name="call">
+  <Call>
+    <Name>call</Name>
```

Comment:
Might want to leave some of these `<Call>` elements still using the attr version, like this one `<Call name="call">`

Reply:
This is a parameterised test, so the same test is done with elements and with attributes.